edmq_backend/src/scripts/generate_song_list.py

```diff
@@ -0,0 +1,34 @@
+import requests
+import json
+import shazam_data
+
+# Youtube API Request
+part = 'snippet'
+playlist_id = 'PLu-qQLnSm9zQwyCVh_c9uXJ6PDbEFBU8P'
+api_key = 'AIzaSyAJtLJ-uNEpBLZqWHngF_Xpji2NmHFsqSQ'
+playlist_url = 'https://youtube.googleapis.com/youtube/v3/playlistItems?part=' + part + '&playlistId=' + playlist_id + '&key=' + api_key
+
+# Load response into dictionary
+yt_response = requests.get(playlist_url).text
+playlist_dict = json.loads(yt_response)
+
+video_id = []
+video_items = playlist_dict['items']
+
+song_list = {}
+song_list['index'] = []
```

Comment:
Does the index of a dictionary need to be a string literal here?

Reply:
and all other places in the file

Reply:
I shouldn't have used the array name "index" for this. I should have used something like "songs" for less confusion. I don't think dictionary keys have to be strings, but for a json file, this usually is a string that specifies the content inside to make parsing easier.